### ISSUE TITLE
How to remove highlighting on the Boxy scoll bars

### ISSUE THREAD

**Issue by mattst** [oivva/st-boxy#118]
Hi,

```
Boxy Theme -> Environment:

- Boxy Theme: 3.4.0
- Sublime Text: 3114
- Platform: linux
- Package Control: True
- Dependencies:
    * mdpopups: 1.10.0
    * markdown: 2.6.6
    * pygments: 2.1a0
    * jinja2: 2.8
```

First of all, Boxy is the best theme ever written, well done, and thank you so much for it !!

I would like to change a small aspect of Boxy Tomorrow regarding the scroll bars background highlighting but I do not know which settings to change.

Firstly I should say that I know how to tweak Boxy and I have made a few small changes by creating a `Packages/User/Boxy Tomorrow.sublime-theme` file to override some of the theme's values. I have only made 6 changes and, just in case you want to see them, I've put them in [this Gist](https://gist.github.com/mattst/c1e3181696658c5c7ef37f4242749d4e).

With the Boxy scroll bars if the ST `"overlay_scroll_bars": "enabled"` setting is used then they look like the screen capture below, when you scroll using the keyboard or with the mouse's scroll wheel then the scroll bars background blends in, there is no highlighting.

![st_boxy_tomorrow_ 1_overlay_scroll_bars_enabled_cursor_not_on_scroll_bar](https://cloud.githubusercontent.com/assets/835623/18320208/c27a21a6-7520-11e6-9727-a037df05134f.png)

With the same `"overlay_scroll_bars": "enabled"` setting but if you use your mouse pointer to drag the scroll bar puck, then the background of the scroll bar being used becomes highlighted; see the vertical scroll bar in the screen capture below.

![st_boxy_tomorrow_ 2_overlay_scroll_bars_enabled_cursor_on_scroll_bar](https://cloud.githubusercontent.com/assets/835623/18320210/c661916e-7520-11e6-80fb-d4a881fb149d.png)

Now if the ST `"overlay_scroll_bars": "disabled"` setting is used, so that the scroll bars are always displayed, then the background of the scroll bar is always highlighted; see both the vertical and horizontal scroll bar in the screen capture below.

![st_boxy_tomorrow_ 3_overlay_scroll_bars_disabled](https://cloud.githubusercontent.com/assets/835623/18320215/ca5b6e3e-7520-11e6-9b09-4d3e91a0882d.png)

I like always having the scroll bars displayed by using the `"overlay_scroll_bars": "disabled"` setting, but I do not like the permenant scroll bar background highlighting. What settings must I add to my `Packages/User/Boxy Tomorrow.sublime-theme` file so that the scroll bars never have any background highlighting even if the mouse pointer is over them? In other words the scroll bars are always shown and always look like this screen capture.

![st_boxy_tomorrow_ 1_overlay_scroll_bars_enabled_cursor_not_on_scroll_bar](https://cloud.githubusercontent.com/assets/835623/18320208/c27a21a6-7520-11e6-9727-a037df05134f.png)

Many thanks.

**Comment by oivva** [comment]
Thanks :wink:  Your request is already in the dev branch and it will be available in 3.5.0 (it will be released in a week). Currently if you want to add it – see d404995.

**Comment by mattst** [comment]
Thanks so much, that is brilliant.

Please note that `"theme_scrollbar_semi_overlayed": true` did not actually work at all when I installed the `d404995` commit - I'm certain I did so correctly and that `"overlay_scroll_bars": "disabled"` was set, and I restarted ST in case that was necessary.

BUT when I installed the latest commit `e98b809` it worked perfectly.

Thanks again.
